client/me/purchases/manage-purchase/purchase-meta.tsx

```diff
@@ -82,6 +82,8 @@ export default function PurchaseMeta( {
 			? translate( 'Renewal Price' )
 			: translate( 'Price' );
 
+	const hideTaxString = isIncludedWithPlan( purchase ) || isOneTimePurchase( purchase );
```

Comment:
Sorry for the confusion, but for the one-time purchases, the goal of #88872 is to hide the entire section (starting at the list item on L112), not just the tax message. It would ultimately look like the following for a Quick Start:

<img width="1063" alt="Screenshot 2024-04-06 at 9 36 22 AM" src="https://github.com/Automattic/wp-calypso/assets/942359/ef41589f-f2cc-4248-b6ae-48716ac0583f">

Reply:
Ah, my apologies - totally missed that. Should be fixed now.